### DISCUSSION TITLE
Do gambling bots break any guidelines?

### DISCUSSION THREAD

**Discussion by NotIGuess** [community/community#23867] [New to GitHub]
<p>Would posting a gambling bot to GitHub (publicly) break any guidelines / risk any actions being taken against me by GitHub?</p>

**Comment by Zerotask** [comment, marked as answer]
<p>With posting you mean creating a repository with the source code? No, it wouldn’t.</p>

**Comment by thespikevolleyball716-ctrl** [comment]
Gambling bots can break guidelines depending on how and where they are used. Most online casinos and betting platforms strictly prohibit automated play, and using bots on these sites can result in account bans, loss of winnings, or permanent suspension. In some regions, gambling bots may also raise legal concerns if they bypass safeguards or manipulate systems. While bots used for simulations, research, or offline analysis are generally acceptable, deploying them on real-money platforms without permission usually violates platform rules and fair-play policies.
